### PR TITLE
Micro Focus Data Protector Template

### PR DESCRIPTION
**Pull request by ramassh**

### CODE REVIEW

File: templates/data-protector-Backup/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+- Pools
+- Processes
+- Sessions
+
+These templates use Data Protector command line to execute commands on the Data Protector platform. Choose the appropraite template .CFG file monitor the Data Protector platform:
+- *dpForWin.cfg* to monitor Windows Data Protector servers
+- *dpForUnix.cfg* to monitor Linux/Unix Data Protector servers
+
+Once appropriate template is imported into a TrueSight CMA policy for Monitoring Studio X (Configuration Variables), you need to:
+- add the Data Protector servers with appropraite *System Credentials* that allow connecting to the server and run Data Protector CLIs
```
Comment (bertysentry) [inline templates/data-protector-Backup/README.md]: typo

File: templates/data-protector-Backup/dpForUnix.cfg [hunk 2]
```diff
@@ -0,0 +1,1142 @@
+PATROL_CONFIG
+"/SENTRY/X/dpForUnix/actionList" = { REPLACE = "" },
+"/SENTRY/X/dpForUnix/credentialsList" = { REPLACE = "" },
+"/SENTRY/X/dpForUnix/databaseType" = { REPLACE = "" },
+"/SENTRY/X/dpForUnix/description" = { REPLACE = "Micro Focus Data Protector Template for Linux/UNIX" },
```
Comment (bertysentry) [inline templates/data-protector-Backup/dpForUnix.cfg]: Please add the required macros in the description, so people know what to put in there:

```md
- **DP_BIN_PATH**: Data Protector binary path. By default this points to %DP_HOME_DIR%/bin on Windows and /opt/omni/bin on Linux/UNIX
- **CLIENT_HOURS**: Number of hours client statistics are searched for. Default: 72 hours
- **SESSION_HOURS**: Number of hours sessions are monitored for. Default: 72 hours
```

File: templates/data-protector-Backup/README.md (hunk 1 above)
Comment (ramassh) [reply templates/data-protector-Backup/README.md]: Fixed

File: templates/data-protector-Backup/dpForUnix.cfg (hunk 2 above)
Comment (ramassh) [reply templates/data-protector-Backup/dpForUnix.cfg]: Fixed.

**Comment by ramassh** [comment]
@bertysentry can you please review and merge?

**Pull request merged**